common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -2896,6 +2911,75 @@ object CelebornConf extends Logging {
       .longConf
       .createOptional
 
+  val WORKER_JVM_QUAKE_ENABLED: ConfigEntry[Boolean] =
+    buildConf("celeborn.worker.jvm.quake.enabled")
+      .categories("worker")
+      .version("0.3.2")
+      .doc("When true, Celeborn worker will start the jvm quake to monitor of gc behavior, " +
+        "which enables early detection of memory management issues and facilitates fast failure.")
+      .booleanConf
+      .createWithDefault(false)
+
+  val WORKER_JVM_QUAKE_CHECK_INTERVAL: ConfigEntry[Long] =
+    buildConf("celeborn.worker.jvm.quake.check.interval")
+      .categories("worker")
+      .version("0.3.2")
+      .doc("Interval of gc behavior checking for worker jvm quake.")
+      .timeConf(TimeUnit.MILLISECONDS)
+      .createWithDefaultString("1s")
+
+  val WORKER_JVM_QUAKE_RUNTIME_WEIGHT: ConfigEntry[Double] =
+    buildConf("celeborn.worker.jvm.quake.runtime.weight")
+      .categories("worker")
+      .version("0.3.2")
+      .doc(
+        "The factor by which to multiply running JVM time, when weighing it against GCing time. " +
+          "\"Deficit\" is accumulated as gc_time - runtime * runtime_weight, and is compared against threshold " +
+          "to determine whether to take action.")
+      .doubleConf
+      .createWithDefault(5)
+
+  val WORKER_JVM_QUAKE_DUMP_THRESHOLD: ConfigEntry[Long] =
+    buildConf("celeborn.worker.jvm.quake.dump.threshold")
+      .categories("worker")
+      .version("0.3.2")
+      .doc("The threshold of heap dump for the maximum GC \"deficit\" which can be accumulated before jvmquake takes action.")
+      .timeConf(TimeUnit.MILLISECONDS)
+      .createWithDefaultString("30s")
+
+  val WORKER_JVM_QUAKE_KILL_THRESHOLD: ConfigEntry[Long] =
+    buildConf("celeborn.worker.jvm.quake.kill.threshold")
+      .categories("worker")
+      .version("0.3.2")
+      .doc("The threshold of system kill for the maximum GC \"deficit\" which can be accumulated before jvmquake takes action.")
+      .timeConf(TimeUnit.MILLISECONDS)
+      .createWithDefaultString("60s")
+
+  val WORKER_JVM_QUAKE_DUMP_ENABLED: ConfigEntry[Boolean] =
+    buildConf("celeborn.worker.jvm.quake.dump.enabled")
+      .categories("worker")
+      .version("0.3.2")
+      .doc("Whether to heap dump for the maximum GC \"deficit\" during worker jvm quake.")
+      .booleanConf
+      .createWithDefault(true)
+
+  val WORKER_JVM_QUAKE_DUMP_PATH: OptionalConfigEntry[String] =
+    buildConf("celeborn.worker.jvm.quake.dump.path")
+      .categories("worker")
+      .version("0.3.2")
+      .doc("The path of heap dump for the maximum GC \"deficit\" during worker jvm quake, " +
+        "which default value is [java.io.tmpdir]/celeborn/jvm-quake/dump/[pid].")
+      .stringConf
+      .createOptional
+
+  val WORKER_JVM_QUAKE_EXIT_CODE: ConfigEntry[Int] =
+    buildConf("celeborn.worker.jvm.quake.exit.code")
```

Comment:
Avoid using `.` to introduce unnecessary new namespace
```suggestion
    buildConf("celeborn.worker.jvmQuake.exitCode")
```